### PR TITLE
Add MLOps category

### PR DESCRIPTION
**Pull request by GabrielBianconi**
Following discussion with @anowell, I'm adding an MLOps category.

I'm adding a project I'm involved with (TensorZero) as well as two other projects I'm familiar with.

There was a small issue with the scraping tool: it skipped entries that only had a Github repository and no crates.io crate name. That has been resolved (`main.rs`).

### CODE REVIEW

**Comment by GabrielBianconi** [comment]
I also had to upgrade `actions/upload-artifact` due to a CI error.

**Comment by GabrielBianconi** [comment]
Same for `actions/download-artifact`

File: _data/crates.yaml [hunk 1]
```diff
@@ -60,7 +60,7 @@
   repository: https://github.com/huggingface/candle
```
Comment (GabrielBianconi) [inline _data/crates.yaml:60]: My VSCode YAML formatter made a few extra changes automatically. Happy to reverse if undesired.

File: _scraper/src/main.rs [hunk 2]
```diff
@@ -57,6 +57,7 @@ async fn main() -> Result<()> {
             .krate
             .as_ref()
             .and_then(|k| k.repository.as_ref().map(|r| Url::parse(r).unwrap()))
+            .or(krate.repository)
```
Comment (GabrielBianconi) [inline _scraper/src/main.rs:60]: Without this line, the scraper skips the `- repository` entries!

**Comment by anowell** [comment]
This all looks great. Merging as is. Thanks for the contribution.

**Pull request merged**